kmelia/kmelia-war/src/main/webapp/kmelia/jsp/publicationLinksManager.jsp

```diff
@@ -26,169 +26,187 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-<title><%=resources.getString("GML.popupTitle")%></title>
-<view:looknfeel />
-<style type="text/css">
-#pubList .selection input {
-	display: block;
-}
-</style>
-<%-- load the css and js file used by tree menu --%>
-<menuTree:head displayCssFile="true" displayJavascriptFile="true" displayIconsStyles="false" contextName="<%=m_context%>"></menuTree:head>
-<%-- personalizable Javascript for  YUI treeView menu --%>
-<script type="text/javascript" src="<%=m_context %>/util/javaScript/treeMenu/menu.js"></script>
-
-<script type="text/javascript">
-var context ='<%=m_context %>';
-var currentNodeId;
-var currentNodeIndex;
-var currentComponent;
-var menuType='<%=MenuConstants.SEE_ALSO_MENU_TYPE%>';
-function buildTree() {
-    //create a new tree:
-    tree = new YAHOO.widget.TreeView("treeDiv1");
-    //turn dynamic loading on for entire tree:
-    tree.setDynamicLoad(loadNodeData, 0);
-    //get root node for tree:
-    var root = tree.getRoot();
-   
-    //add child nodes for tree; our top level nodes are  
-   try{
-   var mes = [];
-   mes =YAHOO.lang.JSON.stringify(<%=TreeHandler.ProcessMenu(request,MenuConstants.SEE_ALSO_MENU_TYPE)%>);
-    mes = YAHOO.lang.JSON.parse(mes);
-   }catch(x){
-      notyError("JSON Parse failed: "+x);
-	  return;
-	}
-   for (var i=0, j=mes.length; i<j; i++) {
-         var tempNode = new YAHOO.widget.TextNode(mes[i], root, false);
-			tempNode.multiExpand =false;
-    } 
-    //render tree with these toplevel nodes; all descendants of these nodes
-    //will be generated as needed by the dynamic loader.
-    tree.draw();
-    //action when a user click on a node
-    tree.subscribe('clickEvent',function(oArgs) { 
-		currentComponent = oArgs.node.data.componentId;
-		// highlight selected node
-		$("#ygtvcontentel"+currentNodeIndex).css({'font-weight':'normal'});
-		setCurrentNodeId(oArgs.node.data.id);
-		currentNodeIndex = oArgs.node.index;
-		$("#ygtvcontentel"+currentNodeIndex).css({'font-weight':'bold'});
+  <title><%=resources.getString("GML.popupTitle")%></title>
+  <view:looknfeel />
+  <style type="text/css">
+    #pubList .selection input {
+      display: block;
+    }
+  </style>
+  <%-- load the css and js file used by tree menu --%>
+  <menuTree:head displayCssFile="true" displayJavascriptFile="true" displayIconsStyles="false" contextName="<%=m_context%>"></menuTree:head>
+  <%-- personalizable Javascript for  YUI treeView menu --%>
+  <script type="text/javascript" src="<%=m_context %>/util/javaScript/treeMenu/menu.js"></script>
+
+  <script type="text/javascript">
+    var context ='<%=m_context %>';
+    var currentNodeId;
+    var currentNodeIndex;
+    var currentComponent;
+    var pubId = '<%=pubId%>';
+    var currentSort = <%=KmeliaPublicationSort.SORT_TITLE_ASC%>;
+    var menuType='<%=MenuConstants.SEE_ALSO_MENU_TYPE%>';
+    function buildTree() {
```

Comment:
All the var here can be replaced either by let or by const (for the constant). This is a strong recommendation of the Javascript rules

Reply:
Done.